apps/web/components/Cookiebar/Cookiebar.vue

```diff
@@ -210,3 +210,10 @@ const getCookiePropertyValue = (cookie: Cookie, propertyKey: string) => {
   return cookie[propertyKey as keyof Cookie]?.toString() || '';
 };
 </script>
+<style>
+@media screen and (max-width: 500px) {
+  .cookieBarHeightForMobile {
+    height: 512.77px;
```

Comment:
While there are rare occasions when we're compelled to write custom CSS due to limitations in Tailwind, this case doesn't fall into that category.